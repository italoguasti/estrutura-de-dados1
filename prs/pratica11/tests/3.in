50
8053025 Arthur
6996094 Arthur
2610279 Miguel
6897681 Helena
3793591 Arthur
7675915 Arthur
0435645 Gabriel
7841334 Alice
8168001 Davi
8362314 Davi
7069456 Gabriel
4995043 Helena
7955335 Isabella
9654894 Pedro
9085304 Miguel
9004471 Bernardo
7911226 Isabella
5861015 Arthur
5294972 Isabella
0021340 Isabella
9795583 Bernardo
7043169 Alice
9893498 Alice
6739891 Alice
8565566 Miguel
6357198 Arthur
5495834 Miguel
5875794 Isabella
1653977 Isabella
6235929 Isabella
2190550 Gabriel
1824168 Helena
4714055 Isabella
1764407 Gabriel
9749257 Arthur
5616825 Pedro
1110473 Davi
6464345 Helena
3324239 Davi
7711074 Arthur
4622012 Helena
3548897 Isabella
5455320 Helena
3803756 Pedro
5772160 Pedro
5418094 Gabriel
1903599 Pedro
5191985 Arthur
8078296 Isabella
8168556 Miguel
