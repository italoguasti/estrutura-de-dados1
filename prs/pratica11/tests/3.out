0021340 Isabella
0435645 Gabriel
1110473 Davi
1653977 Isabella
1764407 Gabriel
1824168 Helena
1903599 Pedro
2190550 Gabriel
2610279 Miguel
3324239 Davi
3548897 Isabella
3793591 Arthur
3803756 Pedro
4622012 Helena
4714055 Isabella
4995043 Helena
5191985 Arthur
5294972 Isabella
5418094 Gabriel
5455320 Helena
5495834 Miguel
5616825 Pedro
5772160 Pedro
5861015 Arthur
5875794 Isabella
6235929 Isabella
6357198 Arthur
6464345 Helena
6739891 Alice
6897681 Helena
6996094 Arthur
7043169 Alice
7069456 Gabriel
7675915 Arthur
7711074 Arthur
7841334 Alice
7911226 Isabella
7955335 Isabella
8053025 Arthur
8078296 Isabella
8168001 Davi
8168556 Miguel
8362314 Davi
8565566 Miguel
9004471 Bernardo
9085304 Miguel
9654894 Pedro
9749257 Arthur
9795583 Bernardo
9893498 Alice
